recipe/meta.yaml

```diff
@@ -40,6 +44,8 @@ requirements:
 test:
   requires:
     - pip
+    # Numba needs NumPy 1.21 or less (according to numba itself)
+    - numpy <1.22.0a0
```

Comment:
This will need to be removed once we fix the numba dependency bounds

Reply:
I assume pip check fails without this. I noticed that too. Sometimes pip finds stricter requirements though because of which numpy wheels are available.

Reply:
`pip check` is not the problem here. The problem is that numba's code actually checks its dependencies at runtime and fails.

Reply:
@cbouss for instance, here is what happens on Windows during testing if NumPy 1.23 is installed in the test environment:
```
File "C:\b\abs_807uh_h4b7\croot\shap_1668647522122\_test_env\lib\site-packages\numba\__init__.py", line 140, in _ensure_critical_deps                                                                                                       
raise ImportError("Numba needs NumPy 1.21 or less")      
```

Reply:
Indeed we need a hotfix here.

Reply:
hotfix PR: https://github.com/AnacondaRecipes/repodata-hotfixes/pull/170